internal/testutil/stringutils.go

```diff
@@ -12,3 +12,14 @@ func GenerateRandomAlphaOnlyString(n int) string {
 	}
 	return string(b)
 }
+
+// InSlice tests whether a string is contained in a slice of strings or not.
+// Comparison is case sensitive
+func InSlice(slice []string, s string) bool {
```

Comment:
I don't think we need our own `testutil` for this. `testify/assert` has `assert.Contains()` which does this already.

The error message we get on failure of `c.Assert(testutil.InSlice(...), checker.Equals, true)` is going to be pretty useless. `Expected true got false` does not help anyone debug a test.

I think we should use `assert.Contains()` instead, which will give a helpful error message. We have already started using these assertions in a few places in the `integration-cli` suite.

Reply:
Good idea 👍  Modified. 🐸

Reply:
and a test need rebuild？